app/assets/javascripts/discourse/app/lib/sidebar/custom-community-section-links.js

```diff
@@ -65,6 +66,14 @@ export function addSectionLink(args, secondary) {
       get title() {
         return args.title;
       }
+
+      get prefixValue() {
+        if (args.icon) {
+          return args.icon;
+        } else {
+          return super.prefixValue;
+        }
+      }
```

Comment:
I would probably write it like this, but up to you:

```suggestion
        return args.icon || super.prefixValue;
      }
```

Reply:
:+1: